resources/ts/Components/Integrations/Detail/CredentialsAuthClients.tsx

```diff
@@ -62,27 +70,32 @@ export const CredentialsAuthClients = ({
       <Heading className="font-semibold lg:min-w-60" level={4}>
         {t("details.credentials.uitid_v2")}
       </Heading>
-      {keyVisibility === KeyVisibility.v1 ? (
-        <div className="flex flex-col flex-1 gap-4">
-          <div>
-            <Trans
-              i18nKey="details.credentials.uitid_alert"
-              components={[
-                <Link
-                  key={t("details.credentials.uitid_alert")}
-                  href={t("details.credentials.uitid_alert_link")}
-                  className="text-publiq-blue-dark hover:underline mb-3"
-                />,
-              ]}
-            />
+      {isKeyVisibilityV1 ? (
+        isKeyVisibilityV1 && !!keyVisibilityUpgrade ? (
```

Comment:
My previous comment still stands here since we check twice for `isKeyVisibilityV1`, since it cannot check the second condition without checking the first. But since it's a plain boolean check it's not costly at runtime so not really major, so good for me either way